docs/CHANGELOG.asciidoc

```diff
@@ -29,6 +30,296 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.5.0]]
+== {kib} 7.5.0
+
+[float]
+[[breaking-7.5.0]]
+=== Breaking changes
+See also <<breaking-changes-7.5, breaking changes in 7.5>>.
+
+Platform::
+* Default to _search instead of _msearch in courier {pull}45174[#45174]
+
+[float]
+[[enhancement-7.5.0]]
+=== Enhancements
+APM::
+* More descriptive page titles {pull}44296[#44296]
+* Removing 0 suffix if array contains only one element {pull}47036[#47036]
+* Changing status code colors on trace summary {pull}47114[#47114]
+* Show `span.http.response.status_code` and `span.http` object in Span details flyout {pull}47322[#47322]
+* Metadata sections should show message when no data is available {pull}47842[#47842]
+* Removing rest_total_hits_as_int and changing hits.total type {pull}47814[#47814]
+* Consistent flyout headers {pull}46312[#46312]
+* Always show transaction breakdown {pull}46684[#46684]
+* JVM List view &amp; JVM metrics page {pull}46779[#46779]
+* Garbage collection metrics charts {pull}47023[#47023]
+* Replace ui/kfetch with core.http {pull}47635[#47635]
+* One-line trace summary {pull}44842[#44842]
+* Documentation and alignment for impact column {pull}47602[#47602]
+* Keywords to select legend color in TPM graph {pull}47628[#47628]
+* Update index pattern {pull}48066[#48066]
+* Increase breadcrumb `max` setting {pull}46595[#46595]
+* Agent configuration GA {pull}46995[#46995]
+Canvas::
+* Share workpads in other websites {pull}46278[#46278]
+Discover::
+* Making filter of fields Lens-like {pull}47847[#47847]
+* Show field type name when hovering over them (#35391) {pull}44208[#44208]
+* De-angularize index pattern selection {pull}46347[#46347]
+* De-angularize side bar search field {pull}46679[#46679]
+* Replace Discover chart with elastic-charts {pull}43788[#43788]
+Graph::
+* Save modal {pull}44261[#44261]
```

Comment:
Removes the current inline-save menu from Graph and switches to a save modal as used in Visualize and Discover